tmtest/tmtest.go

```diff
@@ -37,7 +37,7 @@ func RunTendermint(ctx context.Context, t TestReporter, home string) (cleanup fu
 	tmpath, err := exec.LookPath("tendermint")
 	if err != nil {
 		if os.Getenv("FORCE_TM_TEST") != "1" {
-			t.Skip("Tendermint binary not found. Set FORCE_TM_TEST=1 to fail this test.")
+			t.Skipf("Tendermint binary not found. Set FORCE_TM_TEST=1 to fail this test.")
```

Comment:
This will fail with `go vet`. It must be `t.Skip` as you don't use any arguments.